integrations/extensions/starter-kits/language-model-openai/OpenAI-Extension-action.json

```diff
@@ -0,0 +1,1001 @@
+{
+  "name": "OpenAI Extension-action",
+  "type": "action",
+  "valid": true,
+  "status": "Available",
+  "created": "2023-05-01T15:21:52.252Z",
+  "updated": "2023-05-05T19:06:13.542Z",
+  "language": "en",
+  "skill_id": "a75d51cc-5f4b-4690-9411-6fcd9bfce235",
+  "workspace": {
+    "actions": [
+      {
+        "steps": [
+          {
+            "step": "step_126",
+            "context": {
+              "variables": []
+            },
+            "handlers": [],
+            "resolver": {
+              "type": "callout",
+              "callout": {
+                "path": "/v1/completions",
+                "type": "integration_interaction",
+                "method": "POST",
+                "internal": {
+                  "spec_hash_id": "52abfcce168af45904bbec67aa72171f74fe00fccc025503d374dbe9d83f0898",
+                  "catalog_item_id": "1248907c-8728-4beb-995c-767bb6339336"
+                },
+                "request_mapping": {
+                  "body": [
+                    {
+                      "value": {
+                        "scalar": "text-davinci-003"
```

Comment:
I guess a session variable would be better.  I wouldn't want _this_ action to be interactive, because it is intended to be a subaction for a real use case.  However, it might be nice to have a sample test action that does what you say and sets the value for the variable.  I will do that in this PR or as a fast follow depending on timing.

Reply:
(Done)